spec/spec_helper.rb

```diff
@@ -12,3 +16,11 @@
     c.syntax = :expect
   end
 end
+
+def xmlpp(x)
```

Comment:
Instead of writing `xmlpp`, use the `equivalent-xml` gem to compare XML files. Remove REXML.

e.g.
```
expect(Asciidoctor.convert(input)).to be_equivalent_to(output)
```

Reply:
Done